utils/lib/rr_graph/tests/test_graph.py

```diff
@@ -0,0 +1,772 @@
+#!/usr/bin/env python3
```

Comment:
Any reason not to use (unittest)[https://docs.python.org/3/library/unittest.html] or equivilent?

Reply:
I don't have a particular reason. I can definitely reformat the file, I wanted to make sure I wasn't using an unexpected framework that wasn't wanted in the project.